test/authority_test.go

```diff
@@ -147,3 +151,24 @@ func (s) TestUnixCustomDialer(t *testing.T) {
 		})
 	}
 }
+
+func (s) TestColonPortAuthority(t *testing.T) {
+	port := ":50051"
```

Comment:
We can't hardcode the port as it may already be in use.  The address is in `ss.address` after `Start()`.

Reply:
If I let it pick a port, then how can I set the target to ":[port]" to conduct the test?

Reply:
Am now using a second `clientconn` (dialing the ":[port]" target).